inventory/byo/hosts.localhost

```diff
@@ -0,0 +1,21 @@
+#bare minimum hostfile
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ansible_ssh_user=root
+debug_level=2
+openshift_deployment_type=origin
+openshift_release=3.7
+
+[masters]
+localhost
```

Comment:
Change to:
`localhost ansible_connection=local`
so that ssh auth doesn't have to be configured.